content/the-ember-times-issue-201.md

```diff
@@ -1,28 +1,35 @@
 ---
 title: The Ember Times - Issue No. 201
 authors:
-  - anne-greeth-schot-van-herwijnen
+ - anne-greeth-schot-van-herwijnen
+ - patricklx
 date: 2022-06-27T00:00:00.000Z
 tags:
   - newsletter
   - '2022'
 ---
 
 👋 Emberistas! 🐹
-
+Advanced template support in Intellij Idea.
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [🐹 1. Section title in sentence case](section-url)
+## [💻 1. Advanced template support in Intellij Idea](https://github.com/patricklx/intellij-emberjs-experimental)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+In Intellij Idea the support for Emberjs templates has been quite lacking, Especially compared to vscode (Un)Stable Ember Language Server. With [intellij-emberjs-experimental](https://github.com/patricklx/intellij-emberjs-experimental) I added many features to the original plugin to improve the UX of working/coding with templates. Search for the addon named `Ember Experimental.js` and give it a try. **You will need to disable or remove the other Ember.js plugin to prevent conflicts.**
```

Comment:
Please replace `I` with your name and Github handle so people know it was you :)